File: Leetcode/cpp/896.cpp
class Solution {
public:
    bool isMonotonic(vector<int>& nums) {
        bool increase = true, decrease = true;
        for (int i = 1; i < nums.size(); i++){
            if (nums[i - 1] < nums[i]) {
                decrease = false;
            } else if (nums[i - 1] > nums[i]) {
                increase = false;
            }
        }
        return increase || decrease;
    }
};

File: Leetcode/Python/_896.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        increase = decrease = True
        for i in range(1, len(nums)):
            if nums[i - 1] < nums[i]:
                increase = False
            elif nums[i - 1] > nums[i]:
                decrease = False
        return increase or decrease
        
